src/components/game/graphics/utils/Functions.js

```diff
@@ -0,0 +1,4 @@
+// Linear interpolation function for color gradients
```

Comment:
Make a comment about the distinction between graphic util functions and core util functions, each serve different purposes